[oraclert] selefcm timeout: 8500[oraclert] started
=== RUN   TestServiceDiscoveryUpdate
[oraclert] already started, return existing entry.
-----New Blocking Bug:
---Primitive location:
/fuzz/target/discovery/kubernetes/kubernetes_test.go:96
---Primitive pointer:
0xc00008e5c0
-----End Bug
    kubernetes_test.go:168: timed out, got 0 (max: 2) items, some events are skipped

-----Withdraw prim:0xc00008e5c0
    service_test.go:212: 
        	Error Trace:	kubernetes_test.go:227
        	            				kubernetes_test.go:115
        	            				service_test.go:212
        	Error:      	Not equal: 
        	            	expected: "{\"svc/default/testservice\":{\"Targets\":[{\"__address__\":\"testservice.default.svc:30900\",\"__meta_kubernetes_service_cluster_ip\":\"10.0.0.1\",\"__meta_kubernetes_service_port_name\":\"testport0\",\"__meta_kubernetes_service_port_protocol\":\"TCP\",\"__meta_kubernetes_service_type\":\"ClusterIP\"},{\"__address__\":\"testservice.default.svc:30901\",\"__meta_kubernetes_service_cluster_ip\":\"10.0.0.1\",\"__meta_kubernetes_service_port_name\":\"testport1\",\"__meta_kubernetes_service_port_protocol\":\"UDP\",\"__meta_kubernetes_service_type\":\"ClusterIP\"}],\"Labels\":{\"__meta_kubernetes_namespace\":\"default\",\"__meta_kubernetes_service_annotation_test_annotation\":\"testannotationvalue\",\"__meta_kubernetes_service_annotationpresent_test_annotation\":\"true\",\"__meta_kubernetes_service_label_test_label\":\"testvalue\",\"__meta_kubernetes_service_labelpresent_test_label\":\"true\",\"__meta_kubernetes_service_name\":\"testservice\"},\"Source\":\"svc/default/testservice\"}}"
        	            	actual  : "{}"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-{"svc/default/testservice":{"Targets":[{"__address__":"testservice.default.svc:30900","__meta_kubernetes_service_cluster_ip":"10.0.0.1","__meta_kubernetes_service_port_name":"testport0","__meta_kubernetes_service_port_protocol":"TCP","__meta_kubernetes_service_type":"ClusterIP"},{"__address__":"testservice.default.svc:30901","__meta_kubernetes_service_cluster_ip":"10.0.0.1","__meta_kubernetes_service_port_name":"testport1","__meta_kubernetes_service_port_protocol":"UDP","__meta_kubernetes_service_type":"ClusterIP"}],"Labels":{"__meta_kubernetes_namespace":"default","__meta_kubernetes_service_annotation_test_annotation":"testannotationvalue","__meta_kubernetes_service_annotationpresent_test_annotation":"true","__meta_kubernetes_service_label_test_label":"testvalue","__meta_kubernetes_service_labelpresent_test_label":"true","__meta_kubernetes_service_name":"testservice"},"Source":"svc/default/testservice"}}
        	            	+{}
        	Test:       	TestServiceDiscoveryUpdate
[oraclert]: AfterRun
[oraclert]: AfterRunFuzz
[oraclert]: 2 selects
[oraclert]: CheckBugEnd...
End of unit test. Check bugs

-----NO BLOCKING


-----Withdraw prim:0xc00008e5c0

---Stack:
goroutine 88 [running]:
runtime.DumpAllStack()
	/usr/local/go/src/runtime/myoracle_tmp.go:207 +0x85
gfuzz/pkg/oraclert.AfterRunFuzz(0xc00000d410)
	/usr/local/go/src/gfuzz/pkg/oraclert/oracle.go:369 +0xfd
gfuzz/pkg/oraclert.AfterRun(0xc00000d410)
	/usr/local/go/src/gfuzz/pkg/oraclert/oracle.go:322 +0x73
runtime.Goexit()
	/usr/local/go/src/runtime/panic.go:613 +0x1e5
testing.(*common).FailNow(0xc0003cef00)
	/usr/local/go/src/testing/testing.go:741 +0x3c
github.com/stretchr/testify/require.Equal(0x18eb288, 0xc0003cef00, 0x15312a0, 0xc000532740, 0x15312a0, 0xc000532750, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/stretchr/testify@v1.7.0/require/require.go:165 +0x110
github.com/prometheus/prometheus/discovery/kubernetes.requireTargetGroups(0xc0003cef00, 0xc00028f950, 0xc00048fc20)
	/fuzz/target/discovery/kubernetes/kubernetes_test.go:227 +0x19f
github.com/prometheus/prometheus/discovery/kubernetes.k8sDiscoveryTest.Run(0x18e1ae0, 0xc00015f6c0, 0x0, 0xc00000d518, 0x2, 0xc00028f950, 0xc0003cef00)
	/fuzz/target/discovery/kubernetes/kubernetes_test.go:115 +0x477
github.com/prometheus/prometheus/discovery/kubernetes.TestServiceDiscoveryUpdate(0xc0003cef00)
	/fuzz/target/discovery/kubernetes/service_test.go:212 +0x858
testing.tRunner(0xc0003cef00, 0x17d5908)
	/usr/local/go/src/testing/testing.go:1193 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1238 +0x2b5

goroutine 1 [chan receive]:
testing.(*T).Run(0xc0003cef00, 0x173e1ca, 0x1a, 0x17d5908, 0x4a1101)
	/usr/local/go/src/testing/testing.go:1239 +0x2dc
testing.runTests.func1(0xc0003cec80)
	/usr/local/go/src/testing/testing.go:1511 +0x78
testing.tRunner(0xc0003cec80, 0xc000189c80)
	/usr/local/go/src/testing/testing.go:1193 +0xef
testing.runTests(0xc00000d458, 0x2254580, 0x23, 0x23, 0xc1cc44f2ac84551f, 0x6fce43b21, 0x22694e0, 0x4398a5)
	/usr/local/go/src/testing/testing.go:1509 +0x305
testing.(*M).Run(0xc000294c30, 0x0)
	/usr/local/go/src/testing/testing.go:1417 +0x1eb
go.uber.org/goleak.VerifyTestMain(0x18e2900, 0xc000294c30, 0xc00043fe70, 0x2, 0x2)
	/go/pkg/mod/go.uber.org/goleak@v1.1.10/testmain.go:53 +0x38
github.com/prometheus/prometheus/util/testutil.TolerantVerifyLeak(0xc000294c30)
	/fuzz/target/util/testutil/testing.go:34 +0xd2
github.com/prometheus/prometheus/discovery/kubernetes.TestMain(0xc000294c30)
	/fuzz/target/discovery/kubernetes/kubernetes_test.go:41 +0x57
main.main()
	_testmain.go:113 +0x165

goroutine 90 [chan receive]:
k8s.io/client-go/util/workqueue.(*Type).updateUnfinishedWorkLoop(0xc000282840)
	/go/pkg/mod/k8s.io/client-go@v0.20.5/util/workqueue/queue.go:198 +0xac
created by k8s.io/client-go/util/workqueue.newQueue
	/go/pkg/mod/k8s.io/client-go@v0.20.5/util/workqueue/queue.go:58 +0x135

--- FAIL: TestServiceDiscoveryUpdate (1.02s)
FAIL
